.gitignore

```diff
@@ -1,4 +1,4 @@
 rootfs/opt/router/sbin/router
 rootfs/opt/router/sbin/router.*
 vendor/
-coverage.txt
+coverage.txt
```

Comment:
Looks like this removes the trailing newline in this file. Can we leave that be?